production/helm/loki/Chart.lock

```diff
@@ -0,0 +1,9 @@
+dependencies:
+- name: minio
+  repository: https://helm.min.io/
+  version: 8.0.9
+- name: grafana-agent-operator
```

Comment:
Out of curiosity will this collide if I have the Agent installed already or is Helm supposed to resolve all dependencies?

Reply:
I've read the docs. Should the agent be installed into its own namespace? Maybe we should make this dependency configurable.

Reply:
I think exposing a separate namespace for the agent is a good idea. In the case of someone already having the agent installed, I would suggest they disable this by setting `monitoring.selfMonitoring.grafanaAgent.installOperator` to `false`.